client/containers/containers_defs.cpp

```diff
@@ -90,13 +90,13 @@ QMap<DockerContainer, QString> ContainerProps::containerHumanNames()
 {
     return { { DockerContainer::None, "Not installed" },
              { DockerContainer::OpenVpn, "OpenVPN" },
-             { DockerContainer::ShadowSocks, "OpenVPN over SS" },
+             { DockerContainer::ShadowSocks, "OpenVPN over Shadowsocks" },
```

Comment:
If rename OpenVPN over SS to OpenVPN over Shadowsocks, the interface in this place may look bad again.
![image](https://github.com/user-attachments/assets/824d05c2-924b-45f6-91ed-d9eeb8490b9b)

Reply:
Ok, makes sense! I'll revert this particular change.

Reply:
```suggestion
             { DockerContainer::ShadowSocks, "OpenVPN over SS" },
```